wd_demo.au3

```diff
@@ -86,32 +84,29 @@ Func _WD_Demo()
 			Case $idDebugging
 
 			Case $idButton
-				$bProcess = True
-				ExitLoop
+				RunDemo($idDebugging, $idBrowsers)
 
 			Case Else
 				For $i = 0 To $iCount - 1
 					If $aCheckboxes[$i] = $nMsg Then
 						$aDemoSuite[$i][1] = Not $aDemoSuite[$i][1]
-						ExitLoop
-
+						RunDemo($idDebugging, $idBrowsers)
```

Comment:
This line needs to be removed. Otherwise, the demo routines run every time you toggle one of the checkboxes